parachain/pallets/ethereum-beacon-client/src/benchmarking/mod.rs

```diff
@@ -81,7 +81,7 @@ mod benchmarks {
 		#[extrinsic_call]
 		_(RawOrigin::Signed(caller.clone()), *execution_header_update);
 
-		assert!(<ExecutionHeaders<T>>::contains_key(execution_header_hash));
```

Comment:
This storage item is a header though, unlike BeaconState, so I don't see a good reason to rename it.